### PR TITLE
Add TrustWallet to companies list

### PR DESCRIPTION
**Pull request by maratsubkhankulov**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -525,6 +525,7 @@ Please submit a pull request if you wish to include your company on the list:
 - `blk.io <https://blk.io>`_
 - `comitFS <http://www.comitfs.com/>`_
 - `ConsenSys <https://consensys.net/>`_
+- `TrustWallet <http://trustwalletapp.com>`
```
Comment (conor10) [inline README.rst]: Hi - you're missing the underscore at the end of the URL.

Please can you ensure your entry is in alphabetical order in the list.

Additionally, there's a companies rst file located under docs/source/companies.rst. Please could you include there (appreciate I don't make that clear).

Comment (maratsubkhankulov) [reply README.rst]: sure, thanks for review

**Comment by maratsubkhankulov** [comment]
@conor10

**Comment by maratsubkhankulov** [comment]
@conor10

**Comment by maratsubkhankulov** [comment]
thanks!

**Pull request merged**